docs/conf.py

```diff
@@ -1,133 +1,103 @@
 # -*- coding: utf-8 -*-
-# pylint: disable=invalid-name
-"""
-enterprise documentation build configuration file.
-
-This file is execfile()d with the current directory set to its
-containing dir.
-
-Note that not all possible configuration values are present in this
-autogenerated file.
-
-All configuration values have a default; values that are commented out
-serve to show the default.
-"""
-
-from __future__ import absolute_import, unicode_literals
+#
+# Course Discovery Service documentation build configuration file, created by
+# sphinx-quickstart on Sun Feb 17 11:46:20 2013.
+#
+# This file is execfile()d with the current directory set to its containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
 
+import datetime
 import os
 import sys
-from subprocess import check_call
-import enterprise
 
-import edx_theme
+# on_rtd is whether we are on readthedocs.org
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
 
-import django
-from django.utils import six
-
-# Configure Django for autodoc usage
-django.setup()
+if not on_rtd:  # only import and set the theme if we're building docs locally
+    import edx_theme
+    html_theme = 'edx_theme'
+    html_theme_path = [edx_theme.get_html_theme_path()]
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
-# import sys
 # sys.path.insert(0, os.path.abspath('.'))
+REPO_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(REPO_ROOT)
+
+# # Specify settings module (which will be picked up from the sandbox)
+# os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'settings.devstack')
+
+import django
+from django.conf import settings
+settings.configure()
+django.setup()
 
-# -- General configuration ------------------------------------------------
+# -- General configuration -----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#
 # needs_sphinx = '1.0'
 
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
-extensions = [
-    'edx_theme',
-    'sphinx.ext.autodoc',
-    'sphinx.ext.doctest',
-    'sphinx.ext.intersphinx',
```

Comment:
@mattdrayer intersphinx allows interlinking docs from different artifacts - probably we still need it.